### DISCUSSION TITLE
Is there any way to show encrypted text fields?

### DISCUSSION THREAD

**Discussion by akshdeep-singh** [sqlitebrowser/sqlitebrowser#3589] [Q&A] [answered]
I have a database in which some text fields are encrypted (using DES algorithm). The database itself is NOT encrypted. Is it possible to browse the data with decrypted text?

**Comment by mgrojo** [comment, marked as answer]
With an SQLite extension that provides a function decrypting using DES and using a display format. I'm not aware of that extension, but maybe you can develop one. It's not difficult, if you know C.
